{
  "Header": {
    "Original": "CVE-2019-13484 (In Xymon through 4.3.28, a buffer overflow exists in the status-log vi ...)",
    "Line": 282208,
    "ID": "CVE-2019-13484",
    "Description": "(In Xymon through 4.3.28, a buffer overflow exists in the status-log vi ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-1898-1}",
      "Line": 282209,
      "Type": "xref",
      "Bugs": [
        "DLA-1898-1"
      ]
    },
    {
      "Original": "- xymon 4.3.29-1",
      "Line": 282210,
      "Type": "package",
      "Package": "xymon",
      "Kind": "fixed",
      "Version": "4.3.29-1"
    },
    {
      "Original": "[buster] - xymon 4.3.28-5+deb10u1",
      "Line": 282211,
      "Type": "package",
      "Release": "buster",
      "Package": "xymon",
      "Kind": "fixed",
      "Version": "4.3.28-5+deb10u1"
    },
    {
      "Original": "[stretch] - xymon 4.3.28-2+deb9u1",
      "Line": 282212,
      "Type": "package",
      "Release": "stretch",
      "Package": "xymon",
      "Kind": "fixed",
      "Version": "4.3.28-2+deb9u1"
    },
    {
      "Original": "NOTE: https://lists.xymon.com/archive/2019-July/046570.html",
      "Line": 282213,
      "Type": "NOTE",
      "Description": "https://lists.xymon.com/archive/2019-July/046570.html"
    }
  ]
}